### PR TITLE
Remove blog page from the project

### LINKED ISSUE #172: DROP BLOG_PAGE

### PR DESCRIPTION
**Pull request by mecies**
It caused more problems than it added value, sadly it's time for the blog page to walk away.
Thank you for your service! 🥇 

Additionally, I updated related docs and screenshots not to include the blog page.

Closes #172

### CODE REVIEW

File: src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -16,27 +15,16 @@ export type NavbarProps = BoxProps & {
...
-  if (type === 'mobile') return <NavbarMobile {...mobileProps} {...props} withoutBlogPage={withoutBlogPage} />;
-  if (type === 'tablet') return <NavbarTablet {...tabletProps} {...props} withoutBlogPage={withoutBlogPage} />;
+  if (type === 'mobile') return <NavbarMobile {...mobileProps} {...props} />;
+  if (type === 'tablet') return <NavbarTablet {...tabletProps} {...props} />;
 
-  return <NavbarDesktop {...props} withoutBlogPage={withoutBlogPage} />;
+  return <NavbarDesktop {...props} />;
 };
 
 export const Navbar = (props: NavbarProps) => {
```
Comment (KonradSzwarc) [inline src/components/Navbar/Navbar.tsx]: I think, now there is no need to separate `Navbar` and `NavbarComponent`. I would suggest to move all rendering logic to the `Navbar` and remove the `NavbarComponent`

File: src/components/Navbar/NavbarDesktop.tsx [hunk 2]
```diff
@@ -23,13 +21,13 @@ const useStyles = makeStyles((theme) => ({
...
 
-export const NavbarDesktop = ({ className, withoutBlogPage, ...props }: NavbarDesktopProps) => {
+export const NavbarDesktop = ({ className, ...props }: NavbarDesktopProps) => {
   const classes = useStyles();
 
   return (
     <Box {...props} className={clsx(className, classes.root)}>
       <MenuList variant="menu" className={classes.menu}>
-        {renderNavItems(undefined, withoutBlogPage)}
+        {renderNavItems(undefined)}
```
Comment (KonradSzwarc) [inline src/components/Navbar/NavbarDesktop.tsx]: It's not necessary for this PR, but we should fix the logic of this function so we wouldn't pass `undefined` here.

**Review by KonradSzwarc — DISMISSED** [review]

File: src/components/Navbar/NavbarDesktop.tsx (hunk 2 above)
Comment (mecies) [reply src/components/Navbar/NavbarDesktop.tsx]: I did make the parameter optional so we no longer have to pass undefined 👍

File: src/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (mecies) [reply src/components/Navbar/Navbar.tsx]: Done 😄

**Review by KonradSzwarc — DISMISSED** [review]

**Review by KonradSzwarc — DISMISSED** [review]

**Review by KonradSzwarc — APPROVED** [review]

**Pull request merged**